CMakeLists.txt

```diff
@@ -166,6 +190,15 @@ catkin_package(
 #   PATTERN ".svn" EXCLUDE
 # )
 
+# Install everything in the launch/ directory.
+# This CMake macro can also be used to install data files, etc.
+# See http://wiki.ros.org/catkin/CMakeLists.txt#Installing_roslaunch_Files_or_Other_Resources
+# - gordonb 2017-03-15
+install(DIRECTORY launch/
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
+  PATTERN ".svn" EXCLUDE
```

Comment:
This is me being paranoid. ROS CMake formulas all include this EXCLUDE pattern. The reason is that when you use SVN for version control, it leaves `.svn` folders in every subfolder of the repo. Git does not do this, and manages its internal state in a top-level `.git` folder. I don't expect we will ever use SVN, but the exclude does no harm.

Thoughts?